editor/gui/editor_bottom_panel.cpp

```diff
@@ -40,14 +41,25 @@
 #include "editor/themes/editor_scale.h"
 #include "scene/gui/box_container.h"
 #include "scene/gui/button.h"
+#include "scene/gui/separator.h"
 #include "scene/gui/split_container.h"
 
 void EditorBottomPanel::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_READY: {
+			layout_popup = get_popup();
+		} break;
+
 		case NOTIFICATION_THEME_CHANGED: {
 			pin_button->set_button_icon(get_editor_theme_icon(SNAME("Pin")));
 			expand_button->set_button_icon(get_editor_theme_icon(SNAME("ExpandBottomDock")));
 		} break;
+
+		case NOTIFICATION_PREDELETE: {
+			if (layout_popup && !get_popup()) {
+				memfree(layout_popup);
+			}
+		}
```

Comment:
It looks like this is causing a heap use after free error https://github.com/godotengine/godot/actions/runs/19279142673/job/55126147036?pr=108647#step:24:165

There is only one dock context popup that all of the dock slots share, and its a child of EditorNode gui base, so it doesn't need to be deleted here.